packs/data/extinction-curse-bestiary.db/xulgath-bomber.json

```
@@ -756,7 +754,7 @@
                 },
                 "containerId": null,
                 "description": {
-                    "value": "<p>The @Compendium[pf2e.conditionitems.Sickened]{Sickened} condition from concentrated xulgath bile doesn't improve on its own until the target recovers from the poison. A target who recovers from concentrated xulgath bile is immune to being sickened by this poison for 1 minute</p>\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:22]</p>\n<hr />\n<p><strong>Maximum Duration</strong> 6 rounds</p>\n<p><strong>Stage 1</strong> [[/r {1d10}[poison]]]{1d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (1 round)</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 2} (1 round)</p>\n<p><strong>Stage 3</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 3} (1 round)</p>"
+                    "value": "<p>The @Compendium[pf2e.conditionitems.Sickened]{Sickened} condition from concentrated xulgath bile doesn't improve on its own until the target recovers from the poison. A target who recovers from concentrated xulgath bile is immune to being sickened by this poison for 1 minute;</p>\n<hr />\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:22];</p>\n<p><strong>Maximum Duration</strong> 6 rounds;</p>\n<p><strong>Stage 1</strong> [[/r {1d10}[poison]]]{1d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (1 round);</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 2} (1 round);</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 3} (1 round).</p>"
```

```suggestion
                    "value": "<p>The @Compendium[pf2e.conditionitems.Sickened]{Sickened} condition from concentrated xulgath bile doesn't improve on its own until the target recovers from the poison. A target who recovers from concentrated xulgath bile is immune to being sickened by this poison for 1 minute</p>\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:22]</p>\n<hr />\n<p><strong>Maximum Duration</strong> 6 rounds</p>\n<p><strong>Stage 1</strong> [[/r {1d10}[poison]]]{1d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (1 round)</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 2} (1 round)</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 3} (1 round)</p>"
```
Modus operandi has been to place the hr element after the saving throw

We do not need the semicolons to end each line. Those are used to signify line breaks while saving print space, and we do not have that limitation

---

Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64